app/Http/Controllers/PayrollController.php

```diff
@@ -15,7 +25,15 @@ class PayrollController extends Controller
     public function index()
     {
         //
-        return view('hr.payroll.index');
+        $accounting_system_id = $this->request->session()->get('accounting_system_id');
+        $payrolls = Payroll::where('accounting_system_id', $accounting_system_id)
+            ->orderBy('created_at', 'desc')->get();
+        // get payrolls of this year for selecting period
+        $payrolls_this_year = Payroll::where('accounting_system_id', $accounting_system_id)
+            ->whereYear('created_at', date('Y'))
+            ->orderBy('created_at', 'desc')->get();
+
+        return view('hr.payroll.index', compact('payrolls', 'payrolls_this_year'));
```

Comment:
It is also worth noting that we also manage accounting years in Ethiopian Calendar. You may get the starting date from the `start_date` of the first accounting period and the `end_date` of the last accounting period. You can see it applied in navigation bar ;)